### ISSUE TITLE
Camera snapping

### ISSUE THREAD

**Issue by pettrs** [mlgudi/rl-plugins#6]
Could be a neat feature to have it snap between two camera points in the same camera sequence

For example;
I want to have a clean transition from left to right. then I want it to instantly snap to a different camera position then zoom in
so..
Move from camera angle 1, to camera angle 2, then instantly snap to camera angle 3, and move smoothly to camera angle 4

I did solve the issue by adding a 1ms camera angle in between, but a "cleaner" way would be nice
![image](https://github.com/mlgudi/rl-plugins/assets/75093127/6d10e7b1-6256-45de-ac97-4bb407aa10a7)

**Comment by mlgudi** [comment]
Good idea. I will add a "hold" easing option which simply snaps to the new location and holds the position for the chosen duration. Thanks!
